### ISSUE TITLE
[cloud-tensorflow-mnist]:

### ISSUE THREAD

**Issue by yiyiwang515** [googlecodelabs/feedback#268]
For Step 3, I'm not sure why we need to take care of the cross-entropy loss for test data
